workbench-core/datasets-ui/src/models/Dataset.ts

```diff
@@ -11,3 +11,17 @@ export interface DatasetItem {
   name: string;
   storageName: string;
 }
+
+export interface CreateDatasetForm {
+  name?: string;
+  description?: string;
+  projectId?: string;
```

Comment:
"project" is a concept only known to SWB

Reply:
Suggestion on better name would be appreciated

Reply:
is this suggesting that a dataset is a child to a project?

Reply:
Currently the name is vague. But from UI, yes, this is owning project. While I can definitely rename `projectId` -> `owningProjectId`, I think you might mean more generic name rather than `project`, however I'm not sure what that name would be

Reply:
right, not all datasets will have a parent resource. I like metadata and within metadata can exist a projectId.

Reply:
Updated